### PR TITLE
Solution to problem 1

### PR DESCRIPTION
**Pull request by digbeee**

### CODE REVIEW

File: problem1.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import numpy as np
+threes = np.sum(np.arange(0,1000,3))
+fives = np.sum(np.arange(0,1000,5))
+fifteens = np.sum(np.arange(0,1000,15))
+totals = (threes+fives)-fifteens
```
Comment (matthew-brett) [inline problem1.py]: I would say `totals` is a bit mis-named, in that it is in fact a single total.

**Review by matthew-brett — COMMENTED** [review]
Nice - but you might want to have a look at the standard PEP8 Python style guide for reference : https://pep8.org .   In your case this would mean some changes in the spaces around values and commas, and between lines ...

File: problem1.py [hunk 2]
```diff
@@ -1,6 +1,12 @@
 import numpy as np
```
Comment (matthew-brett) [inline problem1.py:1]: Actually, PEP8 says to put 2 lines between the `import` here and the code.  Minor of course ...

**Comment by digbeee** [comment]
Not sure if I completely understand the 4 space indentation, but I've given it a go. Didn't really tell me how to space it with multiple functions grouped together.

**Review by matthew-brett — COMMENTED** [review]
Nice - I personally would use one line for the np.sum statements, but maybe PEP8 feels differently - I can't remember.

**Comment by matthew-brett** [comment]
Nice - good job.

**Pull request merged**